packages/mdc-floating-label/component.ts

```diff
@@ -22,6 +22,7 @@
  */
 
 import {MDCComponent} from '@material/base/component';
+import {ponyfill} from '@material/dom/index';
```

Comment:
What if you did `import {estimateScrollWidth} from '@material/dom/ponyfill';`? That seems to be how the other packages import ponyfill functions.

Reply:
Done